anyway/parsers/cbs/executor.py

```diff
@@ -989,23 +989,19 @@ def create_provider_code_table():
         db.session.commit()
 
 
-def create_views():
-    db.session.execute("DROP VIEW IF EXISTS involved_markers_hebrew")
-    db.session.execute("DROP VIEW IF EXISTS vehicles_markers_hebrew")
-    db.session.execute("DROP VIEW IF EXISTS markers_hebrew")
-    db.session.execute("CREATE VIEW  markers_hebrew AS " + VIEWS.MARKERS_HEBREW_VIEW)
-    db.session.execute("DROP VIEW IF EXISTS involved_hebrew")
-    db.session.execute("CREATE VIEW involved_hebrew AS " + VIEWS.INVOLVED_HEBREW_VIEW)
-    db.session.execute("DROP VIEW IF EXISTS vehicles_hebrew")
-    db.session.execute("CREATE VIEW vehicles_hebrew AS " + VIEWS.VEHICLES_HEBREW_VIEW)
-    db.session.execute(
-        "CREATE VIEW involved_markers_hebrew AS " + VIEWS.INVOLVED_HEBREW_MARKERS_HEBREW_VIEW
-    )
-    db.session.execute(
-        "CREATE VIEW vehicles_markers_hebrew AS " + VIEWS.VEHICLES_MARKERS_HEBREW_VIEW
-    )
-    db.session.commit()
-    logging.info("Created DB Views")
+def create_tables():
+    with db.get_engine().begin() as conn:
+        conn.execute("TRUNCATE involved_markers_hebrew")
+        conn.execute("TRUNCATE vehicles_markers_hebrew")
+        conn.execute("TRUNCATE vehicles_hebrew")
+        conn.execute("TRUNCATE involved_hebrew")
+        conn.execute("TRUNCATE markers_hebrew")
+        conn.execute("INSERT INTO markers_hebrew " + VIEWS.MARKERS_HEBREW_VIEW)
+        conn.execute("INSERT INTO involved_hebrew " + VIEWS.INVOLVED_HEBREW_VIEW)
+        conn.execute("INSERT INTO vehicles_hebrew " + VIEWS.VEHICLES_HEBREW_VIEW)
+        conn.execute("INSERT INTO vehicles_markers_hebrew " + VIEWS.VEHICLES_MARKERS_HEBREW_VIEW)
+        conn.execute("INSERT INTO involved_markers_hebrew " + VIEWS.INVOLVED_HEBREW_MARKERS_HEBREW_VIEW)
+        logging.info("Created DB Hebrew Tables")
```

Comment:
@ziv17 so these are not efficient queries - since we're querying and inserting over million and even 2 million of rows at once. How do you think should this be improved? insertion in bulks? 
Ideally the truncation and insertion will be done in one transaction - which is a lot to handle - maybe we should add a similar solution to the one you did with the cache.
Either way - it's not a top priority right now - just something to have in mind.